erpnext/stock/doctype/item/item.json

```diff
@@ -1360,14 +1362,24 @@
    "fieldname": "skip_transfer_for_manufacture",
    "fieldtype": "Check",
    "label": "Skip Transfer for Manufacture"
+  },
+  {
+   "fieldname": "column_break_uabq2",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "applicable_commission_item",
+   "fieldtype": "Link",
+   "label": "Applicable Commission Item",
+   "options": "Item"
```

Comment:
should not be part of DocType... add custom field using vehicles domain installer

Reply:
Items DocType should be unchanged